assets/xml/objects/object_zod.xml

```diff
@@ -1,50 +1,72 @@
+        <Texture Name="gTijoEyesClosed" OutName="tijo_eyes_closed" Format="rgba16" Width="32" Height="32" Offset="0x6E50" />
+        <Texture Name="gTijoMouthClosedTex" OutName="tijo_mouth_closed" Format="rgba16" Width="32" Height="32" Offset="0x7650" />
+        <Texture Name="gTijoMouthOpenTex" OutName="tijo_mouth_open" Format="rgba16" Width="32" Height="32" Offset="0x7E50" />
+
+        <!-- <Blob Name="object_zod_Blob_008650" Size="0x2800" Offset="0x8650" /> -->
+
+        <!-- Tijo Drums DisplayLists /> -->
+        <DList Name="gTijoDrumFrameDL" Offset="0xA460" />
+        <DList Name="gTijoRideCymbalDL" Offset="0xA550" />
+        <DList Name="gTijoCrashCymbalDL" Offset="0xA5E0" />
+        <DList Name="gTijoHiHatDL" Offset="0xA670" />
+        <DList Name="gTijoDrum1DL" Offset="0xA700" />
+        <DList Name="gTijoDrum2DL" Offset="0xA8F8" />
+        <DList Name="gTijoDrum3DL" Offset="0xAAF0" />
+        <DList Name="gTijoDrum4DL" Offset="0xACE8" />
+        <DList Name="gTijoDrum5DL" Offset="0xAEE0" />
+        <DList Name="gTijoBassDrumDL" Offset="0xB0D8" />
+
+        <!-- Tijo Drums Textures -->
+        <Texture Name="gTijoDrumFrameTex" OutName="tijo_drum_stands" Format="rgba16" Width="32" Height="32" Offset="0xB348" />
+        <Texture Name="gTijoCymbalHiHatTex" OutName="tijo_cymbal_hihat" Format="rgba16" Width="32" Height="32" Offset="0xBB48" />
+        <Texture Name="gTijoDrumsSkinTex" OutName="tijo_drums_skin" Format="rgba16" Width="8" Height="8" Offset="0xC348" />
+        <Texture Name="gTijoDrumsEyesTex" OutName="tijo_eyes" Format="rgba16" Width="32" Height="32" Offset="0xC3C8" />
+        <Texture Name="gTijoDrumsMouthTex" OutName="tijo_drums_mouth" Format="rgba16" Width="16" Height="16" Offset="0xCBC8" />
+        <Texture Name="gTijoDrumsHairTex" OutName="tijo_drums_hair" Format="rgba16" Width="32" Height="32" Offset="0xCDC8" />
+
+        <!-- Tijo Limbs -->
+        <Limb Name="gTijoLowerBodyChairLimb" Type="Standard" EnumName="TIJO_LIMB_BODY_CHAIR" Offset="0xD5C8" />
+        <Limb Name="gTijoUpperBodyLimb" Type="Standard" EnumName="TIJO_LIMB_UPPER_BODY" Offset="0xD5D4" />
+        <Limb Name="gTijoHeadLimb" Type="Standard" EnumName="TIJO_LIMB_HEAD" Offset="0xD5E0" />
+        <Limb Name="gTijoUpperLeftArmLimb" Type="Standard" EnumName="TIJO_LIMB_UPPER_LEFT_ARM" Offset="0xD5EC" />
+        <Limb Name="gTijoLowerLeftArmLimb" Type="Standard" EnumName="TIJO_LIMB_LOWER_LEFT_ARM" Offset="0xD5F8" />
+        <Limb Name="gTijoLeftHandLimb" Type="Standard" EnumName="TIJO_LIMB_LEFT_HAND_LIMB" Offset="0xD604" />
+        <Limb Name="gTijoUpperRightArmLimb" Type="Standard" EnumName="TIJO_LIMB_UPPER_RIGHT_ARM" Offset="0xD610" />
+        <Limb Name="gTijoLowerRightArmLimb" Type="Standard" EnumName="TIJO_LIMB_LOWER_RIGHT_ARM" Offset="0xD61C" />
+        <Limb Name="gTijoRightHandLimb" Type="Standard" EnumName="TIJO_LIMB_LEFT_RIGHT_LIMB" Offset="0xD628" />
```

Comment:
These should all start with left/right, same with the DLs. We also use "forearm"